src/frontend/komponenter/Fagsak/Vilkårsvurdering/GeneriskVilkår/UtførKnapp.tsx

```diff
@@ -2,24 +2,22 @@ import React from 'react';
 import PennFylt from '../../../../ikoner/PennFylt';
 import Penn from '../../../../ikoner/Penn';
 import { randomUUID } from '../../../../utils/commons';
-import { IVilkårResultat } from '../../../../typer/vilkår';
-import { vilkårFeilmeldingId } from './GeneriskVilkår';
 
 interface IProps {
-    aktiv: boolean;
+    aktiv?: boolean;
+    id: string;
     onClick: () => void;
-    vilkårResultat: IVilkårResultat;
 }
 
-const UtførKnapp: React.FC<IProps> = ({ aktiv, onClick, vilkårResultat }) => {
+const UtførKnapp: React.FC<IProps> = ({ aktiv, id, children, onClick }) => {
     return (
         <button
```

Comment:
Why not use Knapp from nav-frontend?

Reply:
Because the button is so different from nav-frontend.